### ISSUE TITLE
Why don't discriminant models and generative models use convolutional neural networks?

### ISSUE THREAD

**Issue by zhouhao-learning** [eriklindernoren/Keras-GAN#123]
Hello, I use your code to generate RGB images. Here's what I did after 30,000 training sessions:
![29800](https://user-images.githubusercontent.com/24581773/53224500-dc04f580-36af-11e9-8a2a-3990588bb3a8.png)

As you can see, the effect of generation is not very good, and there are many noises, so I think, why don't you use convolutional neural network to construct discriminant model and generation model? Is it not possible?

**Comment by niunno** [comment]
See [https://arxiv.org/pdf/1511.06434.pdf](DCGAN)

**Comment by zhouhao-learning** [comment]
@niunno 
ok

**Comment by zhouhao-learning** [comment]
@niunno 
In addition, the loss function of the discriminant model and the generation model in the code is a binary cross-entropy function, but the optimization function of the discriminant model and the generation model mentioned in the paper is not a binary cross-entropy function. What does this mean??

**Comment by eriklindernoren** [comment]
@zhouhao-learning The negative log-likelihood mentioned in the paper and binary crossentropy used in the implementation are equivalent. Please refer to Google for more information.
